crates/viewer/re_view_time_series/src/view_class.rs

```diff
@@ -393,6 +417,70 @@ impl ViewClass for TimeSeriesView {
 
         let state = state.downcast_mut::<TimeSeriesViewState>()?;
 
+        let line_series = system_output.view_systems.get::<SeriesLinesSystem>()?;
```

Comment:
this class' ui method has become _even_ more scary and hard to follow. Any chance we can split it up a little bit? E.g. have things like reset live in its own method?